src/main/pg/motor.c

```diff
@@ -74,6 +74,10 @@ void pgResetFn_motorConfig(motorConfig_t *motorConfig)
 
     motorConfig->motorPoleCount = 14;   // Most brushes motors that we use are 14 poles
```

Comment:
Why is `motorPoleCount` hardcoded here? Function `pgResetFn_motorConfig` isn't called anywhere?

Reply:
@haslinghuis I think this function is resetting all settings to defaults. Like after flashing the firmware.